packages/react-components/src/molecules/ImageLink.tsx

```diff
@@ -0,0 +1,40 @@
+import { EmotionJSX } from '@emotion/react/types/jsx-namespace';
+import { css } from '@emotion/react';
+import { fern } from '../colors';
+
+interface ImageLinkProps {
+  imgSrc?: string;
+  link?: string;
+  alt?: string;
+  placeholder?: EmotionJSX.Element;
+  containerPropStyle?: React.CSSProperties;
+}
+
+const ImageLink: React.FC<ImageLinkProps> = ({
+  imgSrc,
+  link,
+  alt,
+  placeholder,
+  containerPropStyle,
+}) => {
+  const wrapperStyle = link
+    ? {
+        ...containerPropStyle,
+        ...{
+          transition: ' 100ms ease-in-out, color 100ms ease-in-out',
+          ':hover': {
+            color: fern,
+            opacity: '64%',
+          },
+        },
+      }
+    : containerPropStyle;
+
+  return (
+    <a css={css({ ...wrapperStyle })} href={link}>
```

Comment:
You should be able to use Achor. It does some extxra stuff like opening external links in a new window